pkg/goformatters/gci/internal/config/config.go

```diff
@@ -0,0 +1,106 @@
+package config
+
+import (
+	"sort"
+	"strings"
+
+	"gopkg.in/yaml.v3"
+
+	"github.com/daixiang0/gci/pkg/config"
+	"github.com/daixiang0/gci/pkg/section"
+)
+
+var defaultOrder = map[string]int{
+	section.StandardType:    0,
+	section.DefaultType:     1,
+	section.CustomType:      2,
+	section.BlankType:       3,
+	section.DotType:         4,
+	section.AliasType:       5,
+	section.LocalModuleType: 6,
+}
+
+type Config struct {
+	config.BoolConfig
+	Sections          section.SectionList
+	SectionSeparators section.SectionList
+}
```

Comment:
I think the problem here

Reply:
this is a copy https://github.com/daixiang0/gci/blob/dca258a399bf32ab20c73ab4edb726cfff7224ef/pkg/config/config.go#L32-L36

Reply:
I got it. You don't have `github.com/golangci/golangci-lint/pkg/goformatters/gci/internal/section` used anywhere